guides/v2.3/graphql/payment-methods/payflow-express.md

```diff
@@ -0,0 +1,83 @@
+---
+group: graphql
+title: Express Checkout for other PayPal solutions
+---
+
+Set the payment method code to `payflow_express` to process Express Checkout transactions when the payment method is set to any of the following:
+
+-  Payflow Link
+-  Payflow Pro
+-  Payments Advanced
+-  Payments Pro
+
+PayPal Express Checkout enables customers to pay by credit card or from the security of their personal PayPal accounts. During checkout, the customer is redirected to the secure PayPal site to complete the payment information. The customer is then returned to the store to complete the remainder of the checkout process.
+
+From a GraphQL integration standpoint, this payment method is identical to the [PayPal Express Checkout]({{page.baseurl}}/graphql/payment-methods/payflow-link.html) payment method, with the exception of the payment method `code`.
+
+
+## PayPal Express Checkout workflow
+
+The following diagram shows the workflow for placing an order when `payflow_express` is the specified payment method.
+
+![PayPal Express Checkout sequence diagram]({{site.baseurl}}/common/images/graphql/paypal-express-checkout.svg)
+
+The following steps describe the flow of calls required to complete a typical PayPal Express Checkout authorization. A successful purchase requires that you send three mutations to PayPal, and the buyer must approve the purchase by logging in to PayPal.
+
+{% include graphql/payment-methods/paypal-express-checkout-workflow.md %}
+
+## Additional Payment information
+
+When you set the payment method code to ``payflow_express`` in the `setPaymentMethodOnCart` mutation, you must also specify attributes specific to this payment method in `additional_data` object. These attributes are defined in the  `paypal_express` object:
+
+{% include graphql/payment-methods/paypal-express-checkout-attributes.md %}
+
+## Example setPaymentMethodOnCart mutation
+
+The following example shows the `setPaymentMethodOnCart` mutation constructed for the PayPal Express payment method.
+
+**Request**
+
+```graphql
+mutation {
+  setPaymentMethodOnCart(input: {
+    cart_id: "rMQdWEecBZr4SVWZwj2AF6y0dNCKQ8uH"
+    payment_method: {
+        code: "payflow_express"
+        additional_data: {
+            paypal_express: {
```

Comment:
Here the under additional_data should look like "payflow_express" instead of "paypal_express"